pages/api/school/[id].js

```diff
@@ -8,13 +8,34 @@ export default async function handler(req, res) {
 
     switch (method) {
         case "GET":
-        try {
-            const id = req.query.id;
-            const school = await School.findById(id); /* find all the data in our database */
-            res.status(200).json({ success: true, data: school });
-        } catch (error) {
-            res.status(400).json({ success: false });
-        }
-        break;        
+            try {
+                const id = req.query.id;
+                const school = await School.findById(id); /* find all the data in our database */
+                res.status(200).json({ success: true, data: school });
+            } catch (error) {
+                res.status(400).json({ success: false });
```

Comment:
it's better to send error to client side if error was catch .
like : success: false, error: error.message